messages/open.json

```diff
@@ -1,16 +1,18 @@
 {
-  "description": "open your default scratch org, or another specified org\nTo open a specific page, specify the portion of the URL after \"yourInstance.salesforce.com/\" as --path.\nFor example, specify \"--path lightning\" to open Lightning Experience, or specify \"--path /apex/YourPage\" to open a Visualforce page.\nTo generate a URL but not launch it in your browser, specify --urlonly.",
+  "description": "open your default scratch org, or another specified org\nTo open a specific page, specify the portion of the URL after \"yourInstance.salesforce.com/\" as --path.\nFor example, specify \"--path lightning\" to open Lightning Experience, or specify \"--path /apex/YourPage\" to open a Visualforce page.\nTo generate a URL but not launch it in your browser, specify --urlonly.\nTo open in a specific browser specify the app name according to your OS as \"--browser firefox",
   "examples": [
     "sfdx force:org:open",
     "sfdx force:org:open -u me@my.org",
     "sfdx force:org:open -u MyTestOrg1",
-    "sfdx force:org:open -r -p lightning"
+    "sfdx force:org:open -r -p lightning",
+    "sfdx force:org:open -u me@my.org -b firefox"
   ],
+  "browser": "browser to target",
```

Comment:
Small suggested edit:

browser in which the org opens

(okay?  I'm not thrilled with "in which" tho.  maybe "browser where the org opens"?  Your original "browser to target" seemed to be calling out for a wee more detail.  But maybe it's fine as-is.  Maybe I'm overthinking this.  I'll let you decide.)